packages/api-utils/package.json

```diff
@@ -51,7 +51,8 @@
     "graphql-relay": "^0.9.0",
     "lodash": "^4.17.15",
     "ramda": "^0.28.0",
-    "transliteration": "^2.1.9"
+    "transliteration": "^2.1.9",
+    "simpl-schema": "^1.12.0"
```

Comment:
Can we try this using the latest version of SimpleSchema, i.e. 3.x?

Reply:
Are you referring to use 3.x only in this plugin (api-utils) while keeping 1.12 in others or make the change across all?

Reply:
I was thinking anywhere we are adding new, we should add the current version

Reply:
Since this is causing problems let's just go with the standard version of SimpleSchema and look at going to 3 separately

Reply:
Reverted simpl-schema to 1.12